oagutu/hellobooks_client
# [Ft # 160093645] user profile

### What does this PR do?
Allow for a user to view their profile and change their password

### Description of task to be performed
Addition of a user profile page as well as a form for resetting a user's password.

### How can it be manually tested
Once a user is logged in, they should be able to navigate to their profile page, via the profile link in the dropdown nav menu. On the profile page they should be able to view details of their account.
They should also be able to change their password.

### What are the relevant pivotal tracker stories?
[#160093645: View profile](https://www.pivotaltracker.com/story/show/160093645)

### Screenshots
`User profile page`
<img width="1061" alt="screen shot 2018-09-03 at 16 21 23" src="https://user-images.githubusercontent.com/37480490/44989257-f5957200-af95-11e8-930b-8d7a657e1e3d.png">

`change password  modal form`
<img width="1023" alt="screen shot 2018-09-03 at 16 21 31" src="https://user-images.githubusercontent.com/37480490/44989273-0940d880-af96-11e8-9f00-a7db90b2eb79.png">
